Tests/SentryTests/SentryHubTests.swift

```diff
@@ -392,6 +392,26 @@ class SentryHubTests: XCTestCase {
         // only session init is sent
         XCTAssertEqual(1, fixture.client.captureSessionInvocations.count)
     }
+
+    func testCaptureWithoutIncreasingErrorCount() {
+        let sut = fixture.getSut()
+        sut.startSession()
+        fixture.client.sessionIncrementsErrorCount = false
+        sut.capture(error: fixture.error, scope: fixture.scope).assertIsNotEmpty()
+
+        XCTAssertEqual(1, fixture.client.captureErrorWithSessionInvocations.count)
+        if let errorArguments = fixture.client.captureErrorWithSessionInvocations.first {
```

Comment:
`h`: Sorry, I think I forgot to mention that yesterday on our call, or maybe we didn't touch that topic. If we don't increment the error count, there is not need to send a session update, as the session stays the same. In that case, we should only send the event without the session.

Reply:
I've made the change as we discussed at 14:30 but actually it doesn't change anything - we already didn't send the session update if the event was nil. That's because even though yes we were calling `[self sendEvent:event withSession:session withScope:scope]`, that doesn't do anything at all if there is no event. 

So in the end the most recent commit is just cleaning up, but doesn't change any behavior as nothing needed fixing.